Automattic/wp-calypso
# Remove site picker step from woocommerce-installation

I can't find a way to make the site picker optional in https://github.com/Automattic/wp-calypso/pull/58536 so this PR is to remove it completely and we'll just need to rely on the site slug for all testing.

#### Changes proposed in this Pull Request

* Removes site picker step from woocommerce-installation

#### Testing instructions

Confirm transfer still works as expected

* http://calypso.localhost:3000/start/woocommerce-install/confirm?site=wooptestupgrade.wordpres\s.com

Related to #58229
